<?xml version="1.0" encoding="UTF-8"?>   
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1388</width>
    <height>871</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Génération banque d'images</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="lblClasseCourante">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="WidgetImage" name="lblImage">
        <property name="minimumSize">
         <size>
          <width>1280</width>
          <height>720</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Catégories</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnUndo">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Undo</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btnPrecedent">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="imageSlider">
        <property name="tracking">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblIndFrame">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblNbFrames">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnParam">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string>Paramètres</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSuivant">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
        <property name="shortcut">
         <string>Space</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetImage</class>
   <extends>QLabel</extends>
   <header>widgetImage.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnSuivant</tabstop>
  <tabstop>btnPrecedent</tabstop>
  <tabstop>imageSlider</tabstop>
  <tabstop>btnParam</tabstop>
  <tabstop>btnUndo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
